<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompassMoveWidget</class>
 <widget class="QWidget" name="CompassMoveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonOffsetNorth">
     <property name="text">
      <string>N</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/arrow-alt-circle-up-solid.svg</normaloff>:/resources/arrow-alt-circle-up-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonOffsetEast">
     <property name="text">
      <string>E</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/arrow-alt-circle-left-solid.svg</normaloff>:/resources/arrow-alt-circle-left-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinOffset">
     <property name="suffix">
      <string>&quot;</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonOffsetWest">
     <property name="text">
      <string>W</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/arrow-alt-circle-right-solid.svg</normaloff>:/resources/arrow-alt-circle-right-solid.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonOffsetSouth">
     <property name="text">
      <string>S</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/arrow-alt-circle-down-solid.svg</normaloff>:/resources/arrow-alt-circle-down-solid.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
